backend/keystore/keystore.go

```diff
@@ -39,9 +39,16 @@ type Keystore interface {
 	// This is typically done through a screen on the device or through a paired mobile phone.
 	HasSecureOutput(*signing.Configuration, coin.Coin) (bool, error)
```

Comment:
before we had a nice naming pair with `HasSecureOutput` <-> `OutputAddress()`. Maybe now it should be named `CanVerifyAddress` to match the new name `VerifyAddress`?